pkg/security/utils/proc_common.go

```diff
@@ -91,3 +97,84 @@ func GetFilledProcess(p *process.Process) (*FilledProcess, error) {
 		Cmdline:    cmdLine,
 	}, nil
 }
+
+// GetProcessPidNamespace returns the PID namespace of the given PID
+func GetProcessPidNamespace(pid uint32) (uint64, error) {
+	link, err := os.Readlink(fmt.Sprintf("/proc/%d/ns/pid", pid))
```

Comment:
I think this should make use of the `procPidPath` helper function to get the correct pid path

Reply:
Thanks, fixed in last commit